global_helpers/panther_event_type_helpers.py

```diff
@@ -1,7 +1,16 @@
 # a place to hold event type constants used among many data models, rules, or policies
 ADMIN_ROLE_ASSIGNED = "admin_role_assigned"
 FAILED_LOGIN = "failed_login"
+LOGOUT = "logout"
 MFA_DISABLED = "mfa_disabled"
 SUCCESSFUL_LOGIN = "successful_login"
 USER_ACCOUNT_CREATED = "user_account_created"
 ACCOUNT_CREATED = "account_created"
```

Comment:
This has nothing to do with this PR, but could we add a comment to distinguish what the difference between these are?

Reply:
Yup, good call.